services/player-service/adapter-rest/src/main/java/com/kjeldsen/player/rest/delegate/PlayersDelegate.java

```diff
@@ -35,14 +36,15 @@ public ResponseEntity<Void> createPlayer(CreatePlayerRequest createPlayerRequest
             .age(PlayerAge.of(createPlayerRequest.getAge()))
             .position(PlayerPosition.valueOf(createPlayerRequest.getPosition().name()))
             .points(createPlayerRequest.getPoints())
+            .teamId(TeamId.of("NOTEAM"))
             .build();
         createPlayerUseCase.create(newPlayer);
         return new ResponseEntity<>(HttpStatus.CREATED);
     }
 
     @Override
     public ResponseEntity<List<PlayerResponse>> generatePlayer(GeneratePlayersRequest generatePlayersRequest) {
-        List<Player> players = generatePlayersUseCase.generate(generatePlayersRequest.getNumberOfPlayers());
+        List<Player> players = generatePlayersUseCase.generate(generatePlayersRequest.getNumberOfPlayers(), TeamId.of("NOTEAM"));
```

Comment:
to me this indicates that we have 2 usecases, not one with a fixed id for a non existing team. Please breakdown into 2 usecases and rethink naming... for example generateRandomPlayer and generateRandomPlayerWithTeam usecases

Reply:
Yes, also more easy when extends functionality is needed, so we cannot create dependencies between this cases.